lib/sockets-monitor.js

```diff
@@ -44,24 +44,22 @@ SocketsMonitor.prototype.stop = function () {
 
 SocketsMonitor.prototype._captureSocketStats = function () {
   var sockets = globalAgent.sockets;
-  var request = globalAgent.requests;
-  var key;
-  var action;
+  var requests = globalAgent.requests;
   var pidTag = 'pid:' + process.pid;
-
-  for (key in sockets) {
-    action = this.prefix + '.sockets_open';
-    this.monitor.gauge(action, sockets[key].length, 1, ['target:' + key, pidTag]);
-  }
-
-  for (key in request) {
-    action = this.prefix + '.sockets_pending';
-    this.monitor.gauge(action, request[key].length, 1, ['target:' + key, pidTag]);
+  var tags = [pidTag];
+  var self = this;
+  function gaugeCollection (suffix, collection) {
+    for (var key in collection) {
+      var action = self.prefix + '.' + suffix;
+      self.monitor.gauge(action, collection[key].length, 1, tags.concat('target:' + key));
+    }
```

Comment:
yep, I wanted to be private. Can put that on prototype with `_` prefix. Will you prefer that?